modules/command/dh/dh-litigation-answered-command.js

```diff
@@ -44,7 +46,34 @@ class DHLitigationAnsweredCommand extends Command {
                 return Command.empty();
             }
         }
-        return Command.repeat();
+        return Command.retry();
+    }
+
+    /**
+     * Recover system from failure
+     * @param command
+     * @param err
+     */
+    async recover(command, err) {
```

Comment:
I don't think we should call Command retry for failed command. For example, if for some reason this command throws an error unrelated to litigation, it will throw an error 3 times...